vyper/venom/venom_to_assembly.py

```diff
@@ -157,7 +165,9 @@ def generate_evm(self, no_optimize: bool = False) -> list[str]:
 
                 assert fn.normalized, "Non-normalized CFG!"
 
-                self._generate_evm_for_basicblock_r(asm, fn.entry, StackModel())
+                cleanup_needed = any(map(lambda x: bb_cleanup_needed(x), fn.get_basic_blocks()))
```

Comment:
list comprehensions!
```suggestion
                cleanup_needed = any((bb_cleanup_needed(bb) for bb in fn.get_basic_blocks()))
```